src/core_test/mpas_test_core_streams.F

```diff
@@ -391,6 +391,205 @@ subroutine test_core_streams_test(domain, threadErrs, ierr)
          return
       end if
 
+   end subroutine test_read_write_real_streams
+
+   !***********************************************************************
+   !  Subroutine test_remove_alarm
+   !
+   !> \brief   Tests the functionality of adding and removing alarms from
+   !>          input and output streams in the stream manager.
+   !>
+   !> \details This subroutine creates a stream, adds an alarm to both
+   !>          the stream and the clock, and verifies the correct handling
+   !>          of alarms in different scenarios. It ensures that alarms
+   !>          cannot be removed from the opposite list and can be removed
+   !>          from the correct list. The stream type (input or output)
+   !>          is passed as an argument to test both scenarios.
+   !>
+   !> \param domain      The domain object that contains the stream manager
+   !>                    and clock.
+   !> \param ierr        The error code that indicates the result of the test.
+   !> \param streamType  The type of the stream being tested (input or output).
+   !
```

Comment:
As on line 37, can we add a line of hyphens to terminate these documentation blocks? It would be nice to maintain consistent style.